app/src/main/res/layout/code_editor_hs_asd.xml

```diff
@@ -8,9 +8,9 @@
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
         android:layout_height="56dp"
-        android:background="#ff303030"
         android:gravity="center_vertical"
         android:orientation="horizontal"
+        android:background="?android:attr/colorBackground"
```

Comment:
Do not use `android:background` attribute here.
Because view background by default set to `android:colorBackground`.

Reply:
same